docs/source/deployguide/use_CA.md

```diff
@@ -0,0 +1,284 @@
+
+In certificate naming, it is helpful to use a convention that will help you keep track of whether you are referencing a public certificate or a private key. Given that both have the `.pem` extension, consider the following convention for naming public certs and private keys:
+
+* Rename a public cert from `cert.pem` to something meaningful, like `org1-admin-cert.pem`.
+* Rename a private key from `94u498f9r9fr98t49t345545345_sk` to something meaningful like `org1-admin-key.pem`.
+
+In this convention, the last word in the name before appending the `.pem` extension would be either `cert` or `key` to help you remember which is which.
+
+### Creating the local MSP of a node
+
+While the MSP of an organization serves as the representation of the organization on a channel configuration, the local MSP of a node is a logical collection of parameters that is used, along with other parameters, as part of the creation of a node.
+
+As has been noted above, nodes must be bootstrapped both with enrollment certificates (the public/private key pair that identifies a node) and the TLS certificates that encrypt the communication layer between nodes. This "bootstrapping" happens by listing the location of these certificates in the relevant YAML configuration file that is referenced when creating the node. This means that **the local MSP of a node must be created before the node itself can be created**.
+
+For reference, here is a sample [peer configuration file](https://github.com/hyperledger/fabric/blob/release-2.1/sampleconfig/core.yaml). And here is a sample [ordering node configuration file](https://github.com/hyperledger/fabric/blob/release-2.1/sampleconfig/orderer.yaml).
+
+Note that these configuration files ask for the location of the relevant local MSP folder. For the peer, this is defined through the `mspConfigPath`. For the orderer, it is the `LocalMSPDir`. The folders found in this location will be used to define the local MSP of the node, including the private key the node will use when signing its actions as well as the public key of at least one admin of the node.
+
+The TLS certificates, on the other hand, are defined individually, rather than pointing to a folder, and can be found in the `TLS settings` section of the YAML. This means that TLS certificates do not need to be kept in a strict folder structure like the local MSP (relevant in particular to users who will be using an external CA to generate TLS certificates --- use the sample YAML files as a guide to what these certificates are used for). When you enroll a node with the TLS CA, the generated TLS public key can be found in the `/keystore` folder and the TLS private key can be found in the `/signcerts` folder. When you stand up a node that is enabled for TLS, you need to point to these files from the the relevant fields in the YAML config file.
```

Comment:
extra `the` in the last sentence

Reply:
I think the public/private key in `/keystore` and `/signcerts` might be flipped here